src/features/offer/helpers/useOfferSummaryInfoList/useOfferSummaryInfoList.native.test.tsx

```diff
@@ -30,7 +30,7 @@ describe('useOfferSummaryInfoList', () => {
     ])
   })
 
-  it('should return summaryInfoItems dates when offer stock has date in future', async () => {
+  it('should return summaryInfoItems dates when offer stock has date in future and cinema experience is not available', async () => {
```

Comment:
`should return summaryInfoItems dates when offer stock has date in future and is not cinema offer`